test_scripts/WebEngine/WebEngineProjection/005_OnExitApplication_RESOURCE_CONSTRAINT_unregister.lua

```diff
@@ -40,12 +44,14 @@ end
 -- [[ Scenario ]]
 common.Title("Preconditions")
 common.Step("Clean environment", common.preconditions)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
+    { appSessionId, appHMIType })
```

Comment:
```suggestion
  { appSessionId, appHMIType })
```
Please remove redundant indents

Reply:
@VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/e1b8a7c00153273c0320dc9b9b19fe8f2c9a6410#diff-d8e481ca736c621d5cf917740b5d76f7R48)